test/cpp_extensions/open_registration_extension/pytorch_openreg/__init__.py

```diff
@@ -34,7 +34,29 @@ def __exit__(self, type, value, traceback):
             self.idx = driver.exec("uncheckedSetDevice", self.prev_idx)
             return False
 
-    module.device = device  # type: ignore[misc]
+    def device_count() -> int:
+        return driver.exec("deviceCount")
+
+    def is_available():
+        return True
+
+    def current_device():
+        return torch.accelerator.current_device_index()
+
+    def get_rng_state(device):
+        return torch.empty(4, 4, device="openreg")
```

Comment:
We should get a real state here for this and below work.

Reply:
Thank you, will update it in next PR.